events/interaction/features/projectHelper/project-edit-modal-handler.js

```diff
@@ -0,0 +1,62 @@
+import { MessageFlags } from "discord.js";
+import { displayActiveProjects } from "../../../../utils/displayProjects.js";
+import { renderProjectManager } from "../../../../utils/renderProjectManager.js";
+
+export async function handleProjectEditModal(interaction) {
+  if (!interaction.isModalSubmit()) return;
+  const [customId, projectId] = interaction.customId.split(":");
+  if (customId !== "edit_project_modal") return;
+  if (!projectId) return;
+  if (!interaction.botContext.user) return;
+
+  const { projectService } = interaction.services;
+  const name = interaction.fields.getTextInputValue("name").trim();
+  const description = interaction.fields.getTextInputValue("description").trim();
+  await interaction.deferUpdate({ flags: MessageFlags.Ephemeral });
+
+  try {
+    const { user } = interaction.botContext;
+    const existingProjects = await projectService.listByUser(user.id);
+    if (existingProjects.some((p) => p.name.toLowerCase() === name.toLowerCase())) {
```

Comment:
If I only change the description when submitting the edit modal, I get the ' You already have another project with that name. Please choose a different one.' error message. 
Exclude the current project ID from the duplicate check:
```suggestion
      const isDuplicate = existingProjects.some(
      (p) => p.name.toLowerCase() === name.toLowerCase() && String(p.id) !== String(projectId),
    );
    if (isDuplicate) {
```

Reply:
I thought I fixed this, I need to check I didn't push it somewhere else 😅, getting old and senile clearly hehehe.